src/views/assets/Cloud/Account/components/AssetPanel.vue

```diff
@@ -120,9 +120,6 @@ export default {
           action: 'sync_task', account_id: this.object.id
         }))
       }
-      this.ws.onclose = () => {
-        this.$emit('update:visible', false)
-      }
       this.ws.onerror = () => {
         this.$message.error(this.$tc('ConnectWebSocketError'))
       }
```

Comment:
There are no significant differences between the two versions of the code provided. The logic and structure do not appear to have changed; however, I can provide some potential optimizations:

- Use ES6 features: Replace "export default" with `const defaultModule = ...`,
  which is more readable and also adheres to modern JavaScript practices.

- Use arrow functions instead of regular function expressions where possible. 

- Utilize async/await syntax for asynchronous method calls if necessary but note that it requires stricter promises or using native Promises API in most implementations.

If you need help with these specific enhancements or anything else related to your question, please let me know!